airbyte-integrations/connectors/source-monday/source_monday/dpath_string_extractor.py

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from dataclasses import InitVar, dataclass
+from typing import Any, List, Mapping, Union
+
+import dpath.util
+import requests
+from airbyte_cdk.sources.declarative.decoders.decoder import Decoder
+from airbyte_cdk.sources.declarative.decoders.json_decoder import JsonDecoder
+from airbyte_cdk.sources.declarative.extractors.record_extractor import RecordExtractor
+from airbyte_cdk.sources.declarative.interpolation.interpolated_string import InterpolatedString
+from airbyte_cdk.sources.declarative.types import Config, Record
+from dataclasses_jsonschema import JsonSchemaMixin
+
+
+@dataclass
+class DpathStringExtractor(RecordExtractor, JsonSchemaMixin):
```

Comment:
@roman-yermilov-gl can you add a comment explaining how this behaves differently from the default `DPathExtractor`?

Reply:
Done